news/_posts/2019-09-11-prestashop-in-2019-and-beyond-part-2-pain-points.md

```diff
@@ -197,13 +198,13 @@ As if making every class part of our API was not enough of a problem, a lot of l
 
 This is particularly troublesome in PrestaShop, because once a class member has become public—even if it was a mistake—it becomes part of the public API as soon as it's released... and therefore cannot be refactored until the next major version!
 
-It gets _even worse_ because of the [overrides system][overrides-system], which allows developers to replace any legacy core class with a custom one (violating the [Open–Closed principle](https://en.wikipedia.org/wiki/Open%E2%80%93closed_principle), by the way). Since the overrides system is based on inheritance, and inheritance grants overrides access to protected class members of the extended class, and any accessible code is considered part of the API... then **protected members of legacy core classes can be considered part of the public API as well**, because they are accessible. 
+It gets _even worse_ because of the [Overrides system][overrides-system], which allows developers to replace any legacy core class with a custom one (violating the [Open–Closed principle](https://en.wikipedia.org/wiki/Open%E2%80%93closed_principle), by the way). Since the Overrides system is based on inheritance, and inheritance grants overrides access to protected class members of the extended class, and any accessible code is considered part of the API... then **protected members of legacy core classes can be considered part of the public API as well**, because they are accessible. 
```

Comment:
I cant see what changed 😄

Reply:
Ah, the capital O

Reply:
I used uppercase Os for "Overrides system".

Reply:
Yes, thanks, I spotted it a few moments later 😄